mirror/images.txt

```diff
@@ -7,8 +7,8 @@
 #
 
 # ceph-csi devel
-docker.io/rook/ceph:v1.14.9      rook/ceph:v1.14.9
-quay.io/ceph/ceph:v19            ceph/ceph:v19
```

Comment:
lets remove images required for 3.10 and 3.11 and add for 3.14 and 3.15 supported version? v19 will be required for supported version

Reply:
Sure, done!